ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/forms/CollectionSelection.tsx

```diff
@@ -229,7 +229,7 @@ function CollectionSelection({
                     onFocus={ensureOpen}
                     onBlur={handleBlur}
                     autoComplete="off"
-                    id={id}
+                    id={`${id}-select-typeahead`}
```

Comment:
This `id` looks similar to here, correct? Curious if that might apply elsewhere.
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Components/TypeaheadSelect/TypeaheadSelect.tsx#L164